templates/telemetry/pioneer-study/pioneer-study.1.schema.json

```diff
@@ -0,0 +1,75 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "properties": {
+    "creationDate": {
+      "pattern": "^[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}\\.[0-9]{3}Z$",
+      "type": "string"
+    },
+    "id": {
+      "pattern": "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$",
+      "type": "string"
+    },
+    "payload": {
+      "description": "",
+      "properties": {
+        "encryptedData": {
+          "description": "JOSE/JWE encrypted payload.",
+          "type": "string"
+        },
+        "encryptionKeyId": {
+          "description": "JOSE/JWK key id, e.g. pioneer-20170520.",
+          "type": "string"
+        },
+        "pioneerId": {
+          "description": "Custom pioneer id, must not be Telemetry clientId",
+          "pattern": "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$",
+          "type": "string"
+        },
+        "studyName": {
+          "description": "Name of a particular study. Usually the addon_id.",
+          "maxLength": 100,
+          "minLength": 1,
+          "pattern": "^\\S+$",
+          "type": "string"
+        },
+        "studyVersion": {
+          "description": "Semantic version of the study/addon.",
+          "maxLength": 100,
+          "minLength": 1,
+          "pattern": "^\\S+$",
+          "type": "string"
+        }
+      },
+      "required": [
+        "encryptedData",
+        "encryptionKeyId",
+        "pioneerId",
+        "studyName",
+        "studyVersion"
+      ],
+      "title": "pioneer-study",
+      "type": "object"
+    },
+    "type": {
+      "description": "doc_type, restated",
+      "enum": [
+        "pioneer-study"
+      ],
+      "type": "string"
+    },
+    "version": {
+      "maximum": 1,
```

Comment:
Incorrect version telemetry is currently at 4

Reply:
fixed, thx. didn't know this specified the telemetry version, rather than the version of the individual ping schema